src/libraries/System.Drawing.Primitives/tests/RectangleTests.cs

```diff
@@ -165,6 +165,7 @@ public static void GetHashCodeTest()
         [InlineData(float.MinValue, float.MaxValue, float.MinValue, float.MaxValue)]
         [InlineData(0, 0, 0, 0)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/60585", TestPlatforms.iOS | TestPlatforms.tvOS)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/83394", TestPlatforms.Browser)]
```

Comment:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/83394", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT))]
```

Reply:
Question: I don't recall ever using more than one PlatformDetection in ActiveIssue. When adding multiple PlatformDetection items to the ActiveIssue attribute, does it mean it's all or nothing (and) or any of them (or)?